File: SYJ.Application.Dto/EmpleadoDto.cs
using System;

namespace SYJ.Application.Dto {
    public class EmpleadoDto {
        public long EmpleadoID { get; set; }
        public string Nombres { get; set; }
        public string Apellidos { get; set; }
        public System.DateTime FechaNacimiento { get; set; }
        public SexoDto Sexo { get; set; }
        public int NroCedula { get; set; }
        public EstadoCivileDto EstadoCivile { get; set; }
        public NacionalidadeDto Nacionalidade { get; set; }
        public Nullable<int> NumeroIps { get; set; }
        public Nullable<int> NumeroMjt { get; set; }
        public ProfesioneDto Profesione { get; set; }
        public int CantidadHijos { get; set; }
        #region auxiliares
        public SucursaleDto Sucursale { get; set; }
        public CargoDto Cargo { get; set; }
        public bool TieneIpsSn { get; set; }
        /// <summary>
        /// Propiedad utilizada para determinar si el empleado esta activo
        /// o si ya salio de la empresa
        /// </summary>
        public bool Activo { get; set; }
        #endregion
    }
}

File: SueldosYjornales/Scripts/ngapp/src/app/dtos/empleado-dto.ts
import { SexoDto } from 'app/dtos/sexo-dto';
import { EstadoCivileDto } from 'app/dtos/estado-civile-dto';
import { NacionalidadeDto } from 'app/dtos/nacionalidade-dto';
import { ProfesioneDto } from 'app/dtos/profesione-dto';
import { SucursaleDto } from 'app/dtos/sucursale-dto';
import { CargoDto } from 'app/dtos/cargo-dto';

interface IPermitirAddPropDim {// Con este interfaz se permite que se pueda agregar dinamicamente propiedades
    [key: string]: any
}
export class EmpleadoDto implements IPermitirAddPropDim {
    empleadoID: number;
    nombres: string;
    spellidos: string;
    fechaNacimiento: Date;
    sexo: SexoDto;
    nroCedula: number;
    estadoCivile: EstadoCivileDto;
    nacionalidade: NacionalidadeDto;
    numeroIps: number;
    numeroMjt: number;
    profesione: ProfesioneDto;
    cantidadHijos: number;
    sucursale: SucursaleDto;
    cargo: CargoDto;
    tieneIpsSn: boolean;
    /// <summary>
    /// Propiedad utilizada para determinar si el empleado esta activo
    /// o si ya salio de la empresa
    /// </summary>
    activo: boolean;
    // Propiedad auxiliar
    edad: number;
    // Metodos auxiliares
    static getEdad(fechaNacimiento: any): number {
        const hoy = new Date();
        const cumpleanos = new Date(fechaNacimiento);
        let edad = hoy.getFullYear() - cumpleanos.getFullYear();
        const m = hoy.getMonth() - cumpleanos.getMonth();

        if (m < 0 || (m === 0 && hoy.getDate() < cumpleanos.getDate())) {
            edad--;
        }
        return edad;
    }
}
